package/score-worker/src/main/java/io/cloudslang/schema/ConfValue.java

```diff
@@ -50,6 +50,10 @@ public void register(Element element, ParserContext parserContext){
 	}
 
 	private Object fromString(String value) {
+		String sysValue = System.getProperty("cloudslang.worker." + name);
```

Comment:
"cloudslang.worker." , why worker? ConfValue is not only for the worker part...
maybe "cloudslang.conf." or "cloudslang.env."...